Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -215,8 +215,11 @@ class BrowserViewController: UIViewController {
         applyTheme()
     }
 
-    // TODO: FXIOS-5639 Remove delegate keyboard
-    @objc func didTapUndoCloseAllTabToast(notification: Notification) { }
+    /// If user manually opens the keyboard and press undo the app switch to the last open tab
+    /// because of that we need to leave overlay state
```

Comment:
```suggestion
    /// If user manually opens the keyboard and presses undo, the app switches to the last
    /// open tab, and because of that we need to leave overlay state
```